### ISSUE TITLE
The URL to resolve current APPR tag is invalid

### ISSUE THREAD

**Issue by holger12345** [app-registry/appr#95]
Should be changed from https://api.github.com/repos/app-registry/appr/tags to https://github.com/app-registry/appr/tags in cnr.sh, line 10

**Comment by zonepalms91** [comment]
I don't think this is correct. The current URL is for the api endpoint which returns a json response. 
`{
    "name": "v0.7.4",
    "zipball_url": "https://api.github.com/repos/app-registry/appr/zipball/v0.7.4",
    "tarball_url": "https://api.github.com/repos/app-registry/appr/tarball/v0.7.4",
    "commit": {
      "sha": "1509972b9446590578c7d88b89b53e21a6a1f4dd",
      "url": "https://api.github.com/repos/app-registry/appr/commits/1509972b9446590578c7d88b89b53e21a6a1f4dd"
    },
    "node_id": "MDM6UmVmNjk0OTk0OTE6djAuNy40"
  },`

The URL you posted doesn't give a program a parsable response.
